src/pages/Home/Home.page.jsx

```diff
@@ -1,38 +1,13 @@
-import React, { useRef } from 'react';
-import { Link, useHistory } from 'react-router-dom';
-
-import { useAuth } from '../../providers/Auth';
-import './Home.styles.css';
+import React from 'react';
+// eslint-disable-next-line import/extensions
+import VideoList from '../../components/VideoList';
```

Comment:
I recommend having enabled some eslint rules on and figure it out how to solve it. In this [link](https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/extensions.md#examples) you can find some examples of what shows the error and what doesn't, hope it helps.